lib/cli/src/commands/run.rs

```diff
@@ -89,6 +91,10 @@ pub struct RunWithoutFile {
     #[clap(long = "verbose")]
     pub(crate) verbose: Option<u8>,
 
+    /// Enable coredump generation after a WebAssembly trap.
+    #[clap(long = "coredump-on-trap", value_name = "PATH")]
+    coredump_on_trap: Option<String>,
```

Comment:
This should be a PathBuf.

```suggestion
    #[clap(name = "COREDUMP PATH", long = "coredump-on-trap", parse(from_os_str))]
    coredump_on_trap: Option<PathBuf>,
```

Reply:
Fixed, thanks!